### ISSUE TITLE
PS5 Scrape & PS5BOT not recognized 

### ISSUE THREAD

**Issue by TheManWhoLikesToCode** [VVNoodle/PS5bot#12]
I'm brand new to coding and all of this. This is my first time on GitHub I believe I followed all of the installation steps correctly however when I tried to run the program it says "'ps5bot' is not recognized as an internal or external command,
operable program or batch file." I tried changing the file to you .py, .bat, and .exe but none of that worked. I also am confused are you supposed to run the file in Node or Cmd? anything helps I just want to get a PS5 before the new years. Cheers!

**Comment by mc130h** [comment]
Run it in cmd as administrator 
Change the name to ps5bot.py 
Ensure you’re in the /bin directory of the project before you try to run the ps5bot
When you get to the step that say type ps5bot type ps5bot.py instead 
You’ll also need to type ps5bot.py scrape when you get to that part as well.

**Comment by TheManWhoLikesToCode** [comment]
Thanks for the quick feedback man much appreciated! I will test it and get back to you.

**Comment by TheManWhoLikesToCode** [comment]
Wait I lied. So for  "You’ll also need to type ps5bot.py scrape when you get to that part as well." part when I type ps5bot.py scrape, it runs the ps5bot.py and asks me for my first, last name and so on how do I change that. Should I rename scrape to scrape.py?

**Comment by mc130h** [comment]
No. It’s functioning. That is the setup before it actually starts scraping. Once you’re done entering your info it will launch the browser and start scraping.

**Comment by TheManWhoLikesToCode** [comment]
We're ready to go. Enter the following comand to run the scraper:

      ps5bot scrape

I ran ps5bot.py scrape entered in all my stuff got to the bottom and it said this so then I copied and pasted it but its like a loop.

**Comment by texastoastt** [comment]
This isn't written in python, so adding a .py (or any other) file extension won't help you. Type the following in your cmd window: 

node ps5bot scrape
